test/specs/apexReplayDebugger.e2e.ts

```diff
@@ -122,6 +134,11 @@ describe('Apex Replay Debugger', async () => {
     );
     expect(successNotificationWasFound).toBe(true);
 
+    // Close finder tool
+    await browser.keys(['Escape']);
```

Comment:
Why did you put in three Escapes instead of just one?  Doesn't it take only one Escape to close the finder tool?  Or are you putting in extra escapes in case the first one doesn't do the job?

Reply:
First one gets rid of the notification, second one closes the finder tool and the third one unselects the text previously selected